packages/app/src/common/utils.ts

```diff
@@ -63,7 +68,7 @@ export const finalizeAuthResponse = ({ decodedAuthRequest, authRequest, authResp
       }
     }
     window.close();
-  }, 150);
+  }, 250);
```

Comment:
Why was this increased / How does future code editor know not to decrease? Descriptive `const` would be great.

Reply:
The whole setup is messy and could be improved. I bumped this up as I was debugging something, and I think I didn't even need to. I made #430 to clean up and hopefully remove this whole polling system.